static/js/components/Source.jsx

```diff
@@ -63,6 +70,47 @@ const Source = ({ route }) => {
     );
   }
 
+  const groupBy = (array, key) => {
+    // simple groupby for a given key
+    return array.reduce((result, cv) => {
+      (result[cv[key]] = result[cv[key]] || []).push(
+        cv
+      );
+    return result;
+    }, {});
+  };
+
+  function showClassification () {
+      // Here we compute the most recent non-zero probability class for each taxonomy
+
+      const filteredClasses = source.classifications.filter(i => i.probability > 0)
+      const groupedClasses = groupBy(filteredClasses, 'taxonomy_id')
+      const sortedClasses = [];
+
+      Object.keys(groupedClasses).forEach((item, i) => sortedClasses.push(groupedClasses[item].sort(function(a,b){
+            return a.modified < b.modified ? 1 : -1;
+      })));
+
+      if (sortedClasses.length > 0) {
+         return (
+            <div>
+             <b>Classification: </b>
+             {sortedClasses.map((c, index) => (
+                    <Tooltip key={index} disableFocusListener disableTouchListener title={`taxonomy=${c[0].taxonomy_id} P=${c[0].probability}`}>
```

Comment:
The taxonomy name, rather than the id, should be rendered

Reply:
I would also advocate that we show who classified the thing within the taxonomy within these tooltips

Reply:
I disagree -- it's too much info for a quicklook summary.